rotkehlchen/chain/ethereum/modules/uniswap/v2/common.py

```diff
@@ -262,10 +270,33 @@ def decode_uniswap_like_deposit_and_withdrawals(
         factory_address=factory_address,
         init_code_hash=init_code_hash,
     )
+    underlaying_tokens = [
+        UnderlyingToken(address=token0.evm_address, token_kind=EvmTokenKind.ERC20, weight=FVal(0.5)),  # noqa: E501
+        UnderlyingToken(address=token1.evm_address, token_kind=EvmTokenKind.ERC20, weight=FVal(0.5)),  # noqa: E501
+    ]
+    try:
+        pool_token = get_or_create_evm_token(
+            userdb=database,
+            evm_address=pool_address,
+            chain_id=ChainID.ETHEREUM,
+            token_kind=EvmTokenKind.ERC20,
+            evm_inquirer=ethereum_inquirer,
+            seen=TokenSeenAt(tx_hash=tx_hash),
+            underlying_tokens=underlaying_tokens,
+        )
+        if len(pool_token.underlying_tokens) == 0:
+            pool_token = replace(pool_token, underlying_tokens=underlaying_tokens)
```

Comment:
for readability I would just do `dataclasses.replace()` here. Took me some time to find out what replace this is.